src/Stryker.Core/Stryker.Core.UnitTest/Reporters/JsonReporterTests.cs

```diff
@@ -15,16 +15,11 @@
 
 namespace Stryker.Core.UnitTest.Reporters
 {
-    public class JsonReporterTests
+    public class JsonReporterTests : TestBase
     {
         public JsonReporterTests()
         {
-            var logOptions = new LogOptions
-            {
-                LogLevel = Serilog.Events.LogEventLevel.Fatal,
-                LogToFile = false
-            };
-            ApplicationLogging.ConfigureLogger(logOptions, null);
+            ApplicationLogging.LoggerFactory = new LoggerFactory();
```

Comment:
```suggestion
            ApplicationLogging.LoggerFactory = new LoggerFactory();
```

This should be done by the TestBase right

Reply:
Ah yes, I wrote this before I thought of the testbase... I'll remove it